### PR TITLE
Xeno acid mortar & jaw now show up on map and in hive status

### PR DESCRIPTION
**Pull request by Mantlecrawler**
## About The Pull Request
Adds a minimap icon for acid maw & jaw and makes them show up on hive status
image
image
## Why It's Good For The Game
Quality of life
## Changelog

:cl:
qol: Acid Maw & Jaw now show up on hive status and on minimap
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
McXonalds

Also why did you only make one icon?

**Comment by Mantlecrawler** [comment]
> McXonalds
> 
> Also why did you only make one icon?

I couldnt easily fit a J in for Jaw and i couldnt think of anything else to properly represent it

**Comment by Lumipharon** [comment]
You do know xeno icons are like 80% just recolours of other xeno icons right? Add a second one

**Comment by Mantlecrawler** [comment]
@Lumipharon 
image

File: code/modules/xenomorph/maw.dm [hunk 1]
```diff
@@ -223,6 +223,11 @@
 	for(var/datum/maw_ammo/path AS in maw_options)
 		parsed_maw_options[path] = image(icon='icons/mob/radial.dmi', icon_state=path::radial_icon_state)
 	maw_options = parsed_maw_options
+	LAZYADDASSOC(GLOB.xeno_acid_jaws_by_hive, hivenumber, src)
+
+/obj/structure/xeno/acid_maw/Destroy()
+	GLOB.xeno_acid_jaws_by_hive[hivenumber] -= src
+	. = ..()
```
Comment (Lumipharon) [inline code/modules/xenomorph/maw.dm:230]:
```suggestion
	return ..()
```

File: code/modules/xenomorph/maw.dm [hunk 2]
```diff
@@ -285,7 +290,7 @@
 	bound_width = 64
 	bound_height = 64
 	appearance_flags = TILE_BOUND|PIXEL_SCALE|LONG_GLIDE
-	minimap_icon = "acid jaw"
+	minimap_icon = "acid_jaw"
```
Comment (Lumipharon) [inline code/modules/xenomorph/maw.dm:293]: where's the other one bozo

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/xenomorph/maw.dm (hunk 2 above)
Comment (Mantlecrawler) [reply code/modules/xenomorph/maw.dm:293]: thats jaw, maw is above goober

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**